src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -121,6 +121,16 @@
     <EmbeddedResource Update="LanguageServerResources.resx" GenerateSource="true" />
   </ItemGroup>
 
+  <!--
+    Used by LanguageServer.UnitTests project to be able to copy all the output to its own directory.
+    Only needs to run as part of a real build, so skip it in design time builds to prevent issues.
+  -->
+  <Target Name="GetOutputPath" DependsOnTargets="Build" Returns="$(_LanguageServerOutputPath)" Condition="'$(DesignTimeBuild)' != 'true'">
```

Comment:
Originally I had this return the Output files as an item however the RecursiveDir property seemed to get lost on the caller's end.  =(

Reply:
Removed in favor of using the baked in GetTargetPath target.